### DISCUSSION TITLE
How can i use towny prefix with essentialsxchat?

### DISCUSSION THREAD

**Discussion by TheWeirdCoder** [TownyAdvanced/Towny#6423] [Q&A] [answered]
Hey! So i didnt know if this was a towny or essentials question so ima just post in both! So i was wondering how i can show a town prefix with essentialsxchat? Example: "[Example] Owner Theweirdcoder: hi!"

**Comment by LlmDl** [comment, marked as answer]
The townprefix being the town name?

**Reply by TheWeirdCoder** [reply]
Yeah

**Reply by LlmDl** [reply]
https://github.com/TownyAdvanced/Towny/wiki/Placeholders

You probably want something like `%townyadvanced_town%`.

**Reply by TheWeirdCoder** [reply]
So put that in it like
"'%townyadvanced_town%} {PREFIX}{USERNAME} {MESSAGE}'

**Reply by LlmDl** [reply]
I am unsure about how EssentialsChat would want it formatted, that's something their team can answer for you probably.
